### PR TITLE
qt5: upgrade to v5.15.7-lts-lgpl

### PR DESCRIPTION
**Pull request by fmntf**
Please note that this has been tested only with Yocto hardknott.
I did not test the majority of packages since I only care about qtbase. Use at your own risk!

### CODE REVIEW

**Comment by shr-project** [comment]
There is also qt5-creator failure, possibly caused by this upgrade (but not sure I don't remember the last time I've build qt5-creator)

```
| /OE/build/oe-core/tmp-glibc/work/core2-64-oe-linux/qt5-creator/4.15.2+gitAUTOINC+94d227cd43-r0/git/share/qtcreator/qml/qmlpuppet/qmlprivategate/qmlprivategate_56.cpp: In function 'QmlDesigner::PropertyNameList QmlDesigner::Internal::QmlPrivateGate::propertyNameListForWritableProperties(QObject*, const QmlDesigner::PropertyName&, QObjectList*)':
| /OE/build/oe-core/tmp-glibc/work/core2-64-oe-linux/qt5-creator/4.15.2+gitAUTOINC+94d227cd43-r0/git/share/qtcreator/qml/qmlpuppet/qmlprivategate/qmlprivategate_56.cpp:156:82: error: no matching function for call to 'QQuickDesignerSupportProperties::propertyNameListForWritableProperties(QObject*&, const QmlDesigner::PropertyName&, QObjectList*&)'
|   156 |     return QQuickDesignerSupportProperties::propertyNameListForWritableProperties(object, baseName, inspectedObjects);
|       |            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
| In file included from /OE/build/oe-core/tmp-glibc/work/core2-64-oe-linux/qt5-creator/4.15.2+gitAUTOINC+94d227cd43-r0/git/share/qtcreator/qml/qmlpuppet/qmlprivategate/qmlprivategate_56.cpp:44:
| /OE/build/oe-core/tmp-glibc/work/core2-64-oe-linux/qt5-creator/4.15.2+gitAUTOINC+94d227cd43-r0/recipe-sysroot/usr/include/QtQuick/5.15.7/QtQuick/private/qquickdesignersupportproperties_p.h:93:52: note: candidate: 'static QQuickDesignerSupport::PropertyNameList QQuickDesignerSupportProperties::propertyNameListForWritableProperties(QObject*)'
|    93 |     static QQuickDesignerSupport::PropertyNameList propertyNameListForWritableProperties(QObject *object);
|       |                                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
| /OE/build/oe-core/tmp-glibc/work/core2-64-oe-linux/qt5-creator/4.15.2+gitAUTOINC+94d227cd43-r0/recipe-sysroot/usr/include/QtQuick/5.15.7/QtQuick/private/qquickdesignersupportproperties_p.h:93:52: note:   candidate expects 1 argument, 3 provided
| make[4]: *** [Makefile:2068: .obj/release-shared/qmlprivategate_56.o] Error 1
| make[4]: *** Waiting for unfinished jobs....
| make[4]: Leaving directory '/OE/build/oe-core/tmp-glibc/work/core2-64-oe-linux/qt5-creator/4.15.2+gitAUTOINC+94d227cd43-r0/build/src/tools/qml2puppet/qml2puppet'
```

File: recipes-qt/qt5/qtquick3d_git.bb [hunk 1]
```diff
@@ -21,7 +21,7 @@ FILES:${PN}-qmlplugins += " \
   ${OE_QMAKE_PATH_QML}/QtQuick3D/Helpers/meshes/*.mesh \
 "
 
-SRCREV_qtquick3d = "7a8bdd666c8653f80cdee804e679783a461d5f43"
+SRCREV_qtquick3d = "267f179714eff834f95ce79346d7f1215e94b24f"
 SRCREV_assimp = "8f0c6b04b2257a520aaab38421b2e090204b69df"
```
[the remark this answers is not in the document]
Comment (fmntf) [reply recipes-qt/qt5/qtquick3d_git.bb:25]: thank you!

**Comment by Crucio32000** [comment]
I wanted to let you know that i'm actually using this merge request flawlessly.
I was starting from v5.15.4-lts-lgpl and bumped to this.
I was able to create a new build flawlessly as well as the SDK.
Must be said that, in our build, we use the following components:
- QtMultimedia
- QtWayland (only for debug)
- No QML or QtDeclarative
Thank you guys!

**Comment by shr-project** [comment]
Pushed to master with fixed qtquick3d SRCREV and additional fix for qt5-creator, thanks.

**Pull request closed without merging**